### DISCUSSION TITLE
NJSPC Dashpanel forgetting some settings at power-up

### DISCUSSION THREAD

**Discussion by MaxVonEvil** [tagyoureit/nodejs-poolController#867] [Q&A]
Hi, I have NJSPC running as a Docker container on board my Synology NAS and it's almost working perfect. Except, once a blue moon I have to power down the NAS for one reason or another it seems the dash panel looses it's configuration and looks like this:

![image](https://github.com/tagyoureit/nodejs-poolController/assets/19863636/1bdad1d3-4b62-4b32-bac7-e141e3fdb834)

In the dockercompose.yaml file, I have mapped the configuration files to be stored outside and I've doublechecked the permissions in those folders:

![image](https://github.com/tagyoureit/nodejs-poolController/assets/19863636/1ef047ab-474e-468c-afcf-f1434489da1a)

The state of things after power-up is that I manually have to go to the dashpanel URL, dismiss the error message as shown above, then manually find server, click it and apply.

Is there a way to avoid this behavior?
Thank you!

**Comment by rstrouse** [comment]
Those settings are in the config.json.  Is that in your container? Or is the defaultConfig.json written at startup?

**Reply by MaxVonEvil** [reply]
Hmm, if they are in the container and not mapped out, it would explain the reset. Ill have to check. Thanks for the tip!
